viewer/packages/sector-parser/app/testMaterials.ts

```diff
@@ -42,6 +42,21 @@ export function createGeneralCylinderMaterial(): THREE.ShaderMaterial {
       inverseModelMatrix: {
         value: new THREE.Matrix4()
       },
+      modelMatrix: {
+        value: new THREE.Matrix4()
+      },
+      modelViewMatrix: {
+        value: new THREE.Matrix4()
+      },
+      projectionMatrix: {
+        value: new THREE.Matrix4()
+      },
+      normalMatrix: {
+        value: new THREE.Matrix3()
+      },
+      cameraPosition: {
+        value: new THREE.Vector3()
+      },
```

Comment:
I find that when using uniforms, you should usually initialize them with somewhat sensible values. It also makes intellisense happy, so that I can do `myMaterial.uniforms.modelMatrix` and VSCode will pick it up.